### DISCUSSION TITLE
VDH not honoring YouTube audio language selection

### DISCUSSION THREAD

**Discussion by mreboy32** [aclap-dev/video-downloadhelper#431] [Help and Q&A]
### Your question

Video in question: https://www.youtube.com/watch?v=HwAPLk_sQ3w

Above video has 18 different audio languages available. I have "English original" selected, go to grab the video, it downloads. Go to play it and the audio is definitely not English. I don't know enough about most of the languages available so I can't specifically say which one is being downloaded. 

One other oddity that I noticed while trying to figure this out is that on the initial loading of the youtube page there are 6 hits showing when I click the VDH icon on the toolbar (3 MP4 HLS streams followed by 3 MKV HLS streams). If I refresh the page this then becomes 12 hits. Just for giggles I hit refresh a bunch of times and got this up to 22 x 3 = 66 hits at one point before stopping.

![image](https://github.com/aclap-dev/video-downloadhelper/assets/127180927/769f1ac9-6a53-49ed-bc42-e396c9a6714b)

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 657fe6d5193c4052
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:125.0) Gecko/20100101 Firefox/125.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\MREBoy",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user, like you.

The language thing has been reported before.  Take a look at #389.

Yes, VDH detects the page over again when you reload it.  That's how it's always worked.  There is a little trash can at the bottom of the VDH menu.

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/6990cbed-95e5-43f4-93dc-68c30c56c132)
![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/c0c52c63-7d5a-4f2c-a566-3a247a965135)

If you want clarity, you need to click the trash can before you reload a page.
